internal/addon/manifests/charts/mcoa/charts/tracing/templates/opentelemetrycollector.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.enabled }}
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: spoke-otelcol
+  namespace: spoke-otelcol
```

Comment:
Do we need a separate namespace, couldn't it be `openshift-opentelemetry-operator`?

Reply:
Our general recommendation is to use a different namespace from the operator.

Reply:
Is this the namespace of the collector that gets installed in the spoke cluster?

Reply:
Is there any convention in this addon for ns name of deployed instances?

Reply:
> Is this the namespace of the collector that gets installed in the spoke cluster?

Yes

> Is there any convention in this addon for ns name of deployed instances?

No, AFAIK